### PR TITLE
Docs: Added tiiny.host as hosting & deployment option

### PR DESCRIPTION
**Pull request by elbaretto**
Hey guys,

I just wanted to update the docs to add [tiiny.host](https://tiiny.host) as an option for deploying & hosting.

It's currently an option for Gatsby: https://www.gatsbyjs.org/docs/deploying-to-tiiny-host/

Let me know if there's anything else you need from me. Thanks for reviewing it!

### CODE REVIEW

File: docs/content/en/hosting-and-deployment/hosting-on-tiiny.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+weight: 20
+sections_weight: 20
+draft: false
+toc: true
+aliases: []
+---
+
+## Assumptions
+
+You have completed the [Quick Start][] or have a completed Hugo website ready for deployment.
```
Comment (moorereason) [inline docs/content/en/hosting-and-deployment/hosting-on-tiiny.md:24]: You're missing a link for this.

Comment (elbaretto) [reply docs/content/en/hosting-and-deployment/hosting-on-tiiny.md:24]: Thanks for spotting that. I used the firebase template as a guide and just noticed that the Quick Start link is broken there too: https://github.com/gohugoio/hugo/blob/master/docs/content/en/hosting-and-deployment/hosting-on-firebase.md

Would you like me to update that too?

Comment (elbaretto) [reply docs/content/en/hosting-and-deployment/hosting-on-tiiny.md:24]: In fact its broken in many places, here too: `[Quick Start][]` https://github.com/gohugoio/hugo/edit/master/docs/content/en/hosting-and-deployment/hosting-on-github.md

Comment (elbaretto) [reply docs/content/en/hosting-and-deployment/hosting-on-tiiny.md:24]: I've updated it using references as per the other docs

Comment (elbaretto) [reply docs/content/en/hosting-and-deployment/hosting-on-tiiny.md:24]: Hey @moorereason is there anything else you'd like me to add?

Thanks for the review!

**Comment by elbaretto** [comment]
Hey @moorereason any chance in progressing this? Or please let me know if you would like anything else to be added.

**Comment by elbaretto** [comment]
Hey guys

Hope you're well, just following up on this.

I'm working on a campaign to promote Hugo & Gatsby this week and would like to feature Hugo too. Currently [listed with gatsby](https://www.gatsbyjs.org/docs/deploying-to-tiiny-host/).

Looking forwarding to listing Hugo too. Please let me know if there's any issue.

Thanks again for reviewing this.

@bep @moorereason

**Pull request closed without merging**